### ISSUE TITLE
version 0.8.5 not released

### ISSUE THREAD

**Issue by nulian** [xerions/mariaex#248]
Version 0.8.5 is added to the changelog in the branch but never released. 
It also seems the mysql 8 fixes are never release in the 0.9 branch either.

**Comment by jablan** [comment]
ping @liveforeverx ?

**Comment by liveforeverx** [comment]
Right, I wanted to solve this https://github.com/xerions/mariaex/pull/243#issuecomment-446887426, but didn't got to it atm. It is my last todo before release.

So, will try to do it in nearest future and release.

**Comment by jablan** [comment]
Thanks, that PR is exactly why we need 0.8.5, as we have a big problem with prepared statements overflow.

**Comment by sekiyama58** [comment]
Still not available at Hex? I personally using GitHub version directly to solve prepraed statements overflow issue, but want to move public release version.
